python/katana/analytics/_subgraph_extraction.pyx

```diff
@@ -55,6 +71,12 @@ cdef class SubGraphExtractionPlan(Plan):
 
     @staticmethod
     def node_set() -> SubGraphExtractionPlan:
+        """
+        The node-set algorithm:
+
+        * Given a set of node ids, this algorithm constructs a new sub-graph
```

Comment:
Are those * expected here.

Also, it was a bit confusing it says "given a set of node ids", but there are no arguments